lib/inputs/dovetail-bytes.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const timestamp = require('../timestamp');
+const bigquery = require('../bigquery');
+
+/**
+ * Send dovetail downloads to bigquery
+ */
+module.exports = class DovetailDownloadBytes {
+
+  constructor(records) {
+    this._records = (records || []).filter(r => this.check(r));
```

Comment:
Just looking up some docs on Kinesis:
https://docs.aws.amazon.com/streams/latest/dev/service-sizes-and-limits.html

> GetRecords can retrieve up to 10 MiB of data per call from a single shard, and up to 10,000 records per call. Each call to GetRecords is counted as one read transaction.

Is that about the scale we're expecting here?

Reply:
Looking through the CW logs ... seems like we're currently doing about ~50 records every ~1s.  Lambda is configured with batch size = 400.  Guessing that we're just nowhere near that 400 batch size, so lambdas end up executing with much fewer records.

Reply:
As for how many shards we actually need ... it's complicated!

There are actually 2 streams used by dovetail ... `dovetail-downloads-production` (2 shards) and `dovetail-metrics-production-6s` (6 shards).  Downloads is for downloads (entire file), and impressions for adzerk-type impressions.  Looking through the "Monitoring" in the kinesis console, seems like we're nowhere near the red lines on those.  So no real issue "doubling" the traffic from this lambda.  I think.